apps/judicial-system/backend/migrations/20250206152023-update-case.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const replaceEnum = require('sequelize-replace-enum-postgres').default
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    // Add the new boolean field
+    await queryInterface.addColumn('case', 'is_completed_without_ruling', {
+      type: Sequelize.BOOLEAN,
+      allowNull: true,
+    })
+
+    await queryInterface.sequelize.query(`
+      UPDATE "case"
+      SET "is_completed_without_ruling" = true
+      WHERE "decision" = 'COMPLETED_WITHOUT_RULING'
+    `)
+
+    await queryInterface.sequelize.query(`
+      UPDATE "case"
+      SET "decision" = 'ACCEPTING'
+      WHERE "decision" = 'COMPLETED_WITHOUT_RULING'
+    `)
```

Comment:
The two updates could be done in a single update query, resulting in quicker execution. Please consider merging the two.